src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.Analyzer.cs

```diff
@@ -134,39 +134,15 @@ protected override ITypeSymbol GetSymbolType(SemanticModel semanticModel, ISymbo
             {
                 var selectionOperation = semanticModel.GetOperation(SelectionResult.GetContainingScope());
```

Comment:
selection operation is just for the null checking right?  if so, i would prefer you use `syntaxFacts.GetIOperationRootNode(SelectionResult.GetContainingScope());`